src/EnergyPlus/DXCoils.cc

```diff
@@ -12785,53 +12237,77 @@ Label50: ;
 		// SUBROUTINE INFORMATION:
 		//       AUTHOR         Richard Raustad
 		//       DATE WRITTEN   March 2005
-		//       MODIFIED       na
-		//       RE-ENGINEERED  na
 
 		// PURPOSE OF THIS SUBROUTINE:
 		// This subroutine sets an index for a given DX Coil -- issues error message if that
 		// DX Coil is not a legal DX Coil.
 
-		// METHODOLOGY EMPLOYED:
-		// na
-
-		// REFERENCES:
-		// na
-
 		// Using/Aliasing
 		using InputProcessor::FindItemInList;
 
-		// Locals
-		// SUBROUTINE ARGUMENT DEFINITIONS:
+		if ( GetCoilsInputFlag ) {
+			GetDXCoils();
+			GetCoilsInputFlag = false;
+		}
 
-		// SUBROUTINE PARAMETER DEFINITIONS:
-		// na
+		DXCoilIndex = FindItemInList( DXCoilName, DXCoil );
+		if ( DXCoilIndex == 0 ) {
+			if ( present( SuppressWarning ) ) {
+				//     No warning printed if only searching for the existence of a DX Coil
+			} else {
+				if ( present( ThisObjectType ) ) {
+					ShowSevereError( ThisObjectType + ", GetDXCoilIndex: DX Coil not found=" + DXCoilName );
+				} else {
+					ShowSevereError( "GetDXCoilIndex: DX Coil not found=" + DXCoilName );
+				}
+			}
+			ErrorsFound = true;
+		}
 
-		// INTERFACE BLOCK SPECIFICATIONS
-		// na
+	}
 
-		// DERIVED TYPE DEFINITIONS
-		// na
+	std::string
+	GetDXCoilName(
+		int & DXCoilIndex,
+		bool & ErrorsFound,
+		Optional_string_const ThisObjectType,
+		Optional_bool_const SuppressWarning
+	)
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Richard Raustad
+		//       DATE WRITTEN   May 2017
+
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine gets a name for a given DX Coil -- issues error message if that
+		// DX Coil is not a legal DX Coil.
+
+		// Using/Aliasing
+		using InputProcessor::FindItemInList;
 
-		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-		// na
 		if ( GetCoilsInputFlag ) {
 			GetDXCoils();
 			GetCoilsInputFlag = false;
 		}
 
-		DXCoilIndex = FindItemInList( DXCoilName, DXCoil );
 		if ( DXCoilIndex == 0 ) {
 			if ( present( SuppressWarning ) ) {
 				//     No warning printed if only searching for the existence of a DX Coil
 			} else {
 				if ( present( ThisObjectType ) ) {
-					ShowSevereError( ThisObjectType + ", GetDXCoilIndex: DX Coil not found=" + DXCoilName );
+					ShowSevereError( ThisObjectType + ", GetDXCoilIndex: DX Coil not found " );
 				} else {
-					ShowSevereError( "GetDXCoilIndex: DX Coil not found=" + DXCoilName );
+					ShowSevereError( "GetDXCoilIndex: DX Coil not found " );
```

Comment:
same here, similar lines in function GetDXCoilIndex and GetDXCoilName are getting munged.